data/FPI/data.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Frax Price Index",
+    "symbol": "FPI",
+    "decimals": 18,
+    "description": "The Frax Price Index is a protocol within the Frax ecosystem. The system will adjust every month according to an on-chain Consumer Price Index oracle so that holders of the FPI will increase their dollar-denominated value each month according to the reported CPI increase.",
+    "website": "https://frax.finance/",
+    "twitter": "@fraxfinance",
+    "nobridge": false,
```

Comment:
This needs to be true

Reply:
Sorry for all of the confusion. Should I just do one PR with the 3 new tokens (FPIS, FPI, and sfrxETH) then? I guess rejecting FRAX, FXS, and frxETH PRs will go back to the (correct) status quo.